spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/datagen/GenSortDataGenerator.java

```diff
@@ -209,11 +215,6 @@ public void generate() {
                     "Finished running gensort for scale {} and partition {}",
                     SafeArg.of("scale", scale),
                     SafeArg.of("partitionIndex", partitionIndex));
-            DataGenUtils.uploadFiles(
```

Comment:
also small fix to make sure we are not uploading the same partitions multiple times